pepper-apis/src/main/java/org/broadinstitute/ddp/route/AddProfileRoute.java

```diff
@@ -33,8 +34,7 @@ public Object handle(Request request, Response response, Profile profile) throws
         LOG.info("Creating profile for user with guid {}", userGuid);
 
         if (profile == null) {
```

Comment:
I really don't think this case will happen. `ValidatedJsonInputRoute` will have already error-ed out in that case. Since we're changing the API, can we just remove this block here?